server/core/include/server_utilities.hpp

```diff
@@ -35,6 +36,30 @@ namespace irods
     ///
     /// \since 4.2.9
     auto contains_session_variables(const std::string_view _rule_text) -> bool;
+
+    /// Returns the DNS cache eviction age from server_config.json.
+    ///
+    /// If the configuration file cannot be read or parsed, returns a default of 30 seconds.
+    ///
+    /// \return std::chrono::seconds
+    ///
+    /// \since 4.2.9
+    auto get_dns_cache_eviction_age() noexcept -> std::chrono::seconds;
+
+    /// Returns the hostname cache eviction age from server_config.json.
+    ///
+    /// If the configuration file cannot be read or parsed, returns a default of 30 seconds.
+    ///
+    /// \return std::chrono::seconds
+    ///
+    /// \since 4.2.9
+    auto get_hostname_cache_eviction_age() noexcept -> std::chrono::seconds;
```

Comment:
Is there a reason `get_dns_cache_eviction_age` and `get_hostname_cache_eviction_age` are not part of their respective header files?